teachopencadd/talktorials/T032_compound_activity_proteochemometrics/talktorial.ipynb

```diff
@@ -0,0 +1,1920 @@
+{
```

Comment:
<p>Please double-check but I think you can replace</p><pre class="ql-syntax" spellcheck="false">alignment_file = os.path.join(DATA, 'aligned_sequences.aln-fasta.fasta')
</pre><p>with </p><pre class="ql-syntax" spellcheck="false">alignment_file = Path(DATA / "aligned_sequences.aln-fasta.fasta")
</pre><p> </p><p>Comments</p><ul><li>I am always using " instead of ' because this is the default after running <code>black-nb</code> (nothing you need to do manually).</li><li>This means we can drop the <code>os</code> import.</li><li>Try the next code line - SeqIO might need the file path as string instead of a pathlib object, if so do <code>str(alignment_file)</code>.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/278/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>

Reply:
<p>It works with the change! </p>